kuma/wiki/jinja2/wiki/ckeditor_config.js

```diff
@@ -191,6 +192,37 @@
       CKEDITOR.CTRL + 80
     ] );
 
+    // Configure the word count plugin. It will ignore the following:
+    //
+    // * <div> and <p> blocks with the class "hidden" applied
+    // * <div> blocks with the class "prevnext" (that is, Previous and
+    //   Next buttons at the top and/or bottom of pages)
+
+    config.wordcount = {
+        showParagraphs: true,
```

Comment:
This would be a good place for the comment "Don't set maxCharCount or maxWordCount, unless you install the notification plugin as well"